conf/datasources.ini.sample

```diff
@@ -187,3 +187,33 @@ institution = SierraLib
 format = marc
 dedup = true
 building_mapping = sierra_locations.map
+
+; Sample configuration for GeniePlus REST API
+[GeniePlus]
+type = GeniePlus
+genieplusDatabase = "[api_database_name]"
```

Comment:
This one as well as `genieplusTemplate` have a lowercase p, should be unified.

Reply:
Fixed!